contrib/platform/src/com/sun/jna/platform/win32/Pdh.java

```diff
@@ -0,0 +1,282 @@
+/* Copyright (c) 2015 Goldstein Lyor, All Rights Reserved
+ *
+ * This library is free software; you can redistribute it and/or
+ * modify it under the terms of the GNU Lesser General Public
+ * License as published by the Free Software Foundation; either
+ * version 2.1 of the License, or (at your option) any later version.
+ *
+ * This library is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the GNU
+ * Lesser General Public License for more details.
+ */
+package com.sun.jna.platform.win32;
+
+import java.util.Arrays;
+import java.util.List;
+
+import com.sun.jna.Native;
+import com.sun.jna.Structure;
+import com.sun.jna.platform.win32.BaseTSD.DWORD_PTR;
+import com.sun.jna.platform.win32.WinBase.FILETIME;
+import com.sun.jna.platform.win32.WinDef.DWORDByReference;
+import com.sun.jna.platform.win32.WinDef.LONGLONGByReference;
+import com.sun.jna.platform.win32.WinNT.HANDLE;
+import com.sun.jna.platform.win32.WinNT.HANDLEByReference;
+import com.sun.jna.win32.StdCallLibrary;
+import com.sun.jna.win32.W32APIOptions;
+
+/**
+ * Windows Performance Data Helper (a.k.a. PDH).
+ * @author Lyor Goldstein
+ * @see <A HREF="https://msdn.microsoft.com/en-us/library/windows/desktop/aa373083(v=vs.85).aspx">Performance Counters</A>
+ */
+public interface Pdh extends StdCallLibrary {
+    Pdh INSTANCE = (Pdh) Native.loadLibrary("Pdh",
+            Pdh.class, W32APIOptions.UNICODE_OPTIONS);
+
+
+    /** Maximum counter name length. */
+    public static final int PDH_MAX_COUNTER_NAME = 1024;
+    /** Maximum counter instance name length. */
+    public static final int PDH_MAX_INSTANCE_NAME = 1024;
+    /** Maximum full counter path length. */
+    public static final int PDH_MAX_COUNTER_PATH = 2048;
+    /** Maximum full counter log name length. */
+    public static final int PDH_MAX_DATASOURCE_PATH = 1024;
+
+    /* TODO
```

Comment:
FWIW I think if it's superfluous it should be removed everywhere, but since it's superfluous it's no big deal ;)